introduction.md

```diff
@@ -0,0 +1,2 @@
+hello my name is gallop_king and I like rocks. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.